### PR TITLE
Add get_data_id_by_slug method to Process class

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
cc @jkokosar

File: resolwe/process/tests/processes/python_test.py [hunk 1]
```diff
@@ -75,3 +75,6 @@ class PythonProcess2(process.Process):
     name = "Test Python Process 2"
     version = '0.0.1'
     process_type = 'data:python'
+
+    def run(self, inputs, outputs):
```
Comment (dblenkus) [inline resolwe/process/tests/processes/python_test.py]: It would be nice to make this optional.

File: resolwe/api_urls.py [hunk 2]
```diff
@@ -19,5 +19,5 @@
 
 
 urlpatterns = [  # pylint: disable=invalid-name
-    url(r'^', include(api_router.urls, namespace='resolwe-api')),
+    url(r'^api/', include(api_router.urls, namespace='resolwe-api')),
```
Comment (dblenkus) [inline resolwe/api_urls.py]: This should go into a separate commit.

File: resolwe/process/tests/processes/python_test.py (hunk 1 above)
Comment (mstajdohar) [reply resolwe/process/tests/processes/python_test.py]: I'll add this to the Python process wishlist

File: resolwe/api_urls.py (hunk 2 above)
Comment (mstajdohar) [reply resolwe/api_urls.py]: done
